docker/Dockerfile.xenial_kubectl

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:xenial
```

Comment:
I would prefer to base in on the base image ( like all other docker images in istio ). 

This allows the base to be customized independently (and may not be xenial). Also avoids the apt-get update and few other commands.

Reply:
np.  Do you mean Dockerfile.xenial_debug?  Most dockerfiles in istio use from SCRATCH.